pkgs/applications/misc/opentrack/default.nix

```diff
@@ -36,18 +37,26 @@ in
       owner = "opentrack";
       repo = "opentrack";
       rev = "opentrack-${version}";
-      sha256 = "sha256-8gpNORTJclYUYp57Vw/0YO3XC9Idurt0a79fhqx0+mo=";
+      sha256 = "sha256-C0jLS55DcLJh/e5yM8kLG7fhhKvBNllv5HkfCWRIfc4=";
     };
 
-    nativeBuildInputs = [cmake pkg-config ninja copyDesktopItems];
+    patches = [
+      # https://github.com/opentrack/opentrack/pull/1754
+      ./0001-fix-wine-no-wrapper.patch
+    ];
+
+    nativeBuildInputs = [cmake pkg-config ninja copyDesktopItems wine64];
```

Comment:
is it only x86_64 now ?

Reply:
Yes, as the `wine64` package is only available for x86_64, this package is now limited to x86_64 platform.

I added the `meta.platforms` information.
```nix
meta = with lib; {
  ...
  platforms = platforms.x86_64;
};
```

What are our options to have a multi-platform package? I don't know if it's possible to keep this package free of `wine64` and have a new package with wine support (or have a specific build to enable wine support only on x86_64).

Reply:
> this package is now limited to x86_64 platform

This is obviously not great as users on other platforms wouldn't be able to use it anymore. Luckily there's a solution:

`[ ...other deps ] ++ lib.optionals stdenv.targetPlatform.isx86_64 [ wine64 ]`

Also, are you sure this is a build dependency only, and not runtime? In the second case, it would go under `buildInputs`.

Reply:
Thanks for the review.

> This is obviously not great as users on other platforms wouldn't be able to use it anymore. Luckily there's a solution:
> 
> `[ ...other deps ] ++ lib.optionals stdenv.targetPlatform.isx86_64 [ wine64 ]`

Oh nice, I wasn't aware of this trick.
I used the same trick to add the `-DSDK_WINE=ON` build option accordingly.

> Also, are you sure this is a build dependency only, and not runtime? In the second case, it would go under `buildInputs`.

Yes, this is a runtime dependency as opentrack needs the `wine` command in order to use the Wine output.
Moved the dependency to `buildInputs`.
Also replaced `wine64` with `wineWowPackages.stable` to support 32 and 64bits platforms.